### PR TITLE
RobloxCorporation.RobloxBootstrapper version version-dc9e744f72c94313

### PR DESCRIPTION
**Pull request by lokzz**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/24577)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
**Please suppress**:
image

File: manifests/r/RobloxCorporation/RobloxBootstrapper/version-dc9e744f72c94313/RobloxCorporation.RobloxBootstrapper.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: RobloxCorporation.RobloxBootstrapper
+PackageVersion: version-dc9e744f72c94313
+PackageLocale: en-US
+Publisher: Roblox Corporation
+PublisherUrl: https://Roblox.com
+PublisherSupportUrl: https://en.help.roblox.com
+PrivacyUrl: https://en.help.roblox.com/hc/en-us/articles/115004630823-Roblox-Privacy-and-Cookie-Policy
```
Comment (ItzLevvie) [inline manifests/r/RobloxCorporation/RobloxBootstrapper/version-dc9e744f72c94313/RobloxCorporation.RobloxBootstrapper.locale.en-US.yaml:10]:
```suggestion
```

File: manifests/r/RobloxCorporation/RobloxBootstrapper/version-dc9e744f72c94313/RobloxCorporation.RobloxBootstrapper.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+PackageVersion: version-dc9e744f72c94313
+PackageLocale: en-US
+Publisher: Roblox Corporation
+PublisherUrl: https://Roblox.com
+PublisherSupportUrl: https://en.help.roblox.com
+PrivacyUrl: https://en.help.roblox.com/hc/en-us/articles/115004630823-Roblox-Privacy-and-Cookie-Policy
+Author: Roblox Corporation
+PackageName: Roblox Bootstrapper
+License: Copyright © 2020 Roblox Corporation. All rights reserved.
+ShortDescription: Roblox Bootstrapper
```
Comment (ItzLevvie) [inline manifests/r/RobloxCorporation/RobloxBootstrapper/version-dc9e744f72c94313/RobloxCorporation.RobloxBootstrapper.locale.en-US.yaml:14]:
```
ShortDescription: Roblox Bootstrapper
```
**Please write a better description for this application.**

**Comment by rollingmoai** [comment]
@jacklolidk You can try out [Bloxstrap](https://github.com/pizzaboxer/bloxstrap), which is available in Winget.

**Pull request closed without merging**